components/nimbus/metrics.yaml

```diff
@@ -257,6 +257,19 @@ nimbus_events:
       - project-nimbus@mozilla.com
     expires: never
     disabled: true
+  nimbus_test_event:
```

Comment:
Nit: naming; suggest `finished_launching`, or `is_ready`.

Reply:
So this event is specifically to support validation of holdback experiments — maybe something like `holdback_validation_is_ready`?

Reply:
It's being used for the holdback, but the event itself is pretty generic, it seems. I like `is_ready` or something more generic. I also think the event description can be updated:
"A(n) ~test~ event sent when Nimbus finishes launching."

Reply:
Updated!